content/blog/use-redfish-and-ipv6-to-find-lost-servers.md

```diff
@@ -0,0 +1,475 @@
+# iLOrest example
+ilorest login ilo-lio365g11-1 -u <username> -p password
+ilorest get IPv6Addresses/Address MACAddress Oem/Hpe/IPv6/RFC7217Enabled  \
+        --select EthernetInterface.                                       \
+        --filter Name="Manager Dedicated*" --json
+
+{
+  "IPv6Addresses": [
+    {
+      "Address": "FE80::5EED:8CFF:FE01:D7C"
+    }
+  ],
+  "MACAddress": "5C:ED:8C:01:0D:7C",
+  "Oem": {
+    "Hpe": {
+      "IPv6": {
+        "RFC7217Enabled": false
+      }
+    }
+  }
+}
+ilorest logout
+```
+
+```shell
+# Same example with cURL
+curl --noproxy \* --insecure --silent --location --user <ilo-user>:password  \
+     https://ilo-lio365g11-1/redfish/v1/Managers/1/EthernetInterfaces/1 |    \
+     jq '{"MACAddress":.MACAddress, "IPv6Address":.IPv6Addresses[].Address, "RFC7217Enabled": .Oem.Hpe.IPv6.RFC7217Enabled}'
+
+{
+  "MACAddress": "5C:ED:8C:01:0D:7C",
+  "IPv6Address": "FE80::5EED:8CFF:FE01:D7C",
+  "RFC7217Enabled": false
+}
+```
+
+The following example retrieves the same properties as in the previous
+example, but against an iLO with the `RFC7217Enabled` property set
+to `true`. In this example, the link local address does not contain
+the `FF:FE` string and the identifier has no similarities with the MAC address.
+
+```shell
+ilorest login ilo-lio365g11-2 -u <username> -p password
+ilorest get IPv6Addresses/Address MACAddress Oem/Hpe/IPv6/RFC7217Enabled  \
+        --select EthernetInterface.                                       \
+        --filter Name="Manager Dedicated*" --json
+{
+  "IPv6Addresses": [
+    {
+      "Address": "FE80::8751:5A51:F30F:F676"
+    }
+  ],
+  "MACAddress": "5C:ED:8C:01:0E:BE",
+  "Oem": {
+    "Hpe": {
+      "IPv6": {
+        "RFC7217Enabled": true
+      }
+    }
+  }
+}
+ilorest logout
+```
+
+### How to completely disable IPv6 in iLO?
+
+This question is a bit aside of the core subject of this blog post, but, as it is
+a recurrent question and there is enough information to answer it clearly in this blog post, I'll take my chance.
+
+The answer is: *Disable DHCPv6 as well as SLAAC* as shown in Figure 1 below.
+I can hear you reply that you did that, but you still can see the link local address
+like in Figure 1 below. This is expected.
+
+As said previously, SLAAC is always enabled in the iLO OS kernel, and cannot be disabled.
+As a consequence, the IPv6 link local SLAAC address is always present in both the iLO Graphical User Interface (GUI),
+and Redfish responses. However, you will not find any SLAAC addresses resulting of an
+<a href="https://radvd.litech.org/" target="_blank">advertised prefix</a> or a DHCP server.
+
+![Figure 2: SLAAC link local visible when SLAAC disabled](/img/fig2-ipv6-link-local.png "Figure 2: SLAAC link local visible when SLAAC disabled")
+
+<figcaption>Figure 2: SLAAC link local visible when SLAAC disabled</figcaption>
+
+### How to compute an EUI-64 link local
+
+The easiest method to compute the EUI-64 link local address using a MAC address,
```

Comment:
Take out the comma just after "address" and again, take out the comma after "Linux systems"